bridge/steam/steam.go

```diff
@@ -98,11 +98,13 @@ func (b *Bsteam) Send(msg config.Message) (string, error) {
 
 func (b *Bsteam) getNick(id steamid.SteamId) string {
 	b.RLock()
-	defer b.RUnlock()
-	if name, ok := b.userMap[id]; ok {
-		return name
+	name, ok := b.userMap[id]
```

Comment:
why those changes ? what's wrong with the current code ?

Reply:
this is just style to ditch the defer.